api/src/main/java/io/minio/MinioClient.java

```diff
@@ -764,6 +781,61 @@ public MinioClient(
     this.region = region;
   }
 
+  /**
+   * copied logic from
+   * https://github.com/square/okhttp/blob/master/samples/guide/src/main/java/okhttp3/recipes/CustomTrust.java
+   */
+  private static OkHttpClient enableExternalCertiicates(OkHttpClient httpClient, String filename)
```

Comment:
Can you fix the typo here @balamurugana ?

Reply:
done